docs/DatasetEvaluation.md

```diff
@@ -0,0 +1,211 @@
+
+> You'll want to change `tb_log_dir`, `checkpoint_dir` to point to a location that is convenient for you and your Kubernetes cluster have permissions to write to. This is typically a GCS path under the same GCP project. Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to these files inside a docker container.
+
+![pipeline graph](images/kubeflow/pipeline_graph.png)
+
+To open tensorboard for training visualization, you can run the following command:
+
+```bash
+docker run -p 6006:6006 -v $HOME/.config:/root/.config:ro tensorflow/tensorflow tensorboard --host=0.0.0.0 --logdir=gs://<tb_log_dir>
+```
+
+This assumes you have already [logged in](https://cloud.google.com/sdk/gcloud/reference/auth/login) to GCP using a service account stored in the default location, such as `$HOME/.config`. This service account should have permissions to read `tb_log_dir` to download tensorboard files.
+
+Next, follow the [instructions](#part-3-evaluate-a-model) to evaluate evaluate the performance of this model by running one more pipeline we have prepared. You'll need the location of your model in the next step.
+
+## Part 3: Evaluate a model
+
+In this section, we'll a trained model to generate predictions on a held out test set of real images and measure its performance using well-known performance metrics like [mAP](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.evaluation_metrics.html#datasetinsights.evaluation_metrics.average_precision_2d.MeanAveragePrecisionAverageOverIOU) and [mAR](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.evaluation_metrics.html#datasetinsights.evaluation_metrics.average_recall_2d.MeanAverageRecallAverageOverIOU). We have prepared another kubeflow [pipeline](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/evaluate_the_model.yaml) for this.
+
+Simply import the [pre-compiled pipeline](https://raw.githubusercontent.com/Unity-Technologies/datasetinsights/master/kubeflow/compiled/evaluate_the_model.yaml) into your kubeflow cluster. The figure below shows how to do this using the web UI. You can optionally use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
+
+![upload pipeline](images/kubeflow/upload_pipeline.png)
+
+Once your pipeline has been imported, you can run it vis the web UI as shown below. Alternatively, you can use the [KFP CLI Tool](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool)
+
+![evaluate the model](images/kubeflow/evaluate_the_model.png)
+
+Whether you trained a model on synthetic, real or multiple datasets, you'll need to specify a model stored in the `checkpoint_dir` from previous pipelines.
```

Comment:
Can you tell users how to select a model (e.g. select the best model based on validation loss)?

Reply:
The selection of "select the best model based on validation loss" is already covered in various training sections. Here we simply want to focus on "evaluate a model" which is independent of how they pick a model.